test/fixtures/compound16.css

```diff
@@ -0,0 +1,22 @@
+
+:root {
+	--color: #fff000;
+}
+
+#foo li {
+	color: var(--color);
+}
+
+#foo:hover
+{
+	--color: #000000;
+}
+
+#foo div ul li:hover {
+	--color: #aaaaaa;
+}
+
+/* no selector created */
```

Comment:
Let's improve this comment.

```
/* no selector created because it's targeting `a` and the rule with the color declaration is targeting `li` */
```